### DISCUSSION TITLE
How can i use gpu to publish on windows platform

### DISCUSSION THREAD

**Discussion by huangfuhao** [AvaloniaUI/Avalonia#9447] [Q&A] [answered]
When i debug the application , it use gpu to graphic, but it only use cpu to graphic  after publish.
I Overwrite deps.json file in publish floder with deps.json file in debug floder, then copy runtimes to publish floder, 
it is worked , application use gpu again, how can i set application with gpu or say init with gl to publish

![image](https://user-images.githubusercontent.com/116789821/201943408-1df06907-fcd4-4aeb-99b1-0964ce8a08d7.png)

copy json to deps.json in publish floder
 "Avalonia.Angle.Windows.Natives/2.1.0.2020091801": {
        "runtimeTargets": {
          "runtimes/win-arm64/native/av_libglesv2.dll": {
            "rid": "win-arm64",
            "assetType": "native",
            "fileVersion": "2.1.0.0"
          },
          "runtimes/win7-x64/native/av_libglesv2.dll": {
            "rid": "win7-x64",
            "assetType": "native",
            "fileVersion": "2.1.0.0"
          },
          "runtimes/win7-x86/native/av_libglesv2.dll": {
            "rid": "win7-x86",
            "assetType": "native",
            "fileVersion": "2.1.0.0"
          }
        }
      },

**Comment by maxkatz6** [comment, marked as answer]
App should be published with "win10-x64" or "win7-x64" runtime identifier.
